### PR TITLE
BSFER-368 Waiting list additional fields

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: website_event_waiting_list/models/crm_lead.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from odoo import models
+from odoo import models, fields
 
 
 class CrmLead(models.Model):
 
     _inherit = 'crm.lead'
 
+    attendees_number = fields.Integer()
```
Comment (Tonow-c2c) [inline website_event_waiting_list/models/crm_lead.py:10]:
```suggestion
    attendees_number = fields.Integer(string='Number of interested')
```

**Review by Tonow-c2c — COMMENTED** [review]
Please add history line :-)

**Review by i-vyshnevska — APPROVED** [review]
have no objection

**Comment by simahawk** [comment]
> Please add history line :-)

there's no history here ;)

**Pull request merged**